README.md

```diff
@@ -1,7 +1,51 @@
 # bsadd-home
 A static version with some information about the group and it's activities
 
```

Comment:
As a fresh dev, how do I setup this repo? what program needs to be installed, what command need to run to see output of my changes?

Reply:
I'll update accordingly